docs/reference/components/sinks/aws_cloudwatch_logs.cue

```diff
@@ -87,26 +84,26 @@ components: sinks: aws_cloudwatch_logs: {
 	configuration: {
 		create_missing_group: {
 			common:      true
-			description: "Dynamically create a [log group][urls.aws_cloudwatch_logs_group_name] if it does not already exist. This will ignore `create_missing_stream` directly after creating the group and will create the first stream."
+			description: "Dynamically create a [log group](\(urls.aws_cloudwatch_logs_group_name)) if it does not already exist. This will ignore `create_missing_stream` directly after creating the group and will create the first stream."
```

Comment:
I had trouble parsing this sentence out. Perhaps re-word? Maybe re-arrange the order of the 2nd sentence?